src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -58,6 +58,8 @@ public void RunApplication(Application app)
         {
             try
             {
+                AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
```

Comment:
What kind of exceptions will this handle that are unhandled/uncaught by the main try-catch in `RunApplication`?

Reply:
AppDomain.CurrentDomain.UnhandledException catches all unhandled exceptions on all threads (not just UI thread)
ex. new Thread(() => thrown new Exception()).Start()
Unhandled exceptions in user created threads.

main try-catch in RunApplication will not catch all thread unhandled exceptions

Reply:
I did not add it in DynamoVIewModel because if it sometimes causes double CER reports (caught both by Dynamo and REvit). I am investigating this scenario with the CER team.

The double CER report issue happens with the CurrentDispatcher_UnhandledException handler sometimes too. But I am hoping not as much...